### DISCUSSION TITLE
Access Denied

### DISCUSSION THREAD

**Discussion by FloatingBlob** [jef/streetmerchant#2666] [Help]
Access Denied
You don't have permission to access "http://shop.nvidia.com/en-gb/geforce/store/gpu/?" on this server.
Reference #18.7c2b2817.1625826678.1b02a9

I can go to the website on my browser but not chromium for some reason.
Having a vpn doesn't really change anything
Anyone know why
![image](https://user-images.githubusercontent.com/81901681/125065740-1b2e1880-e0aa-11eb-860e-ac6cb68c88a6.png)
also every time I start the script only the first time works, every other time I try to go on the website error 403 pops up

**Comment by Mr-Tech-13** [comment]
Try a proxy, or a VPN
